postgresql/database_test.go

```diff
@@ -1914,6 +1913,224 @@ func TestOptionTypeJsonbStruct(t *testing.T) {
 	}
 }
 
+func TestQueryBuilder(t *testing.T) {
+	var sess db.Database
+	var err error
+	var res db.Result
+	var artist artistType
+
+	if sess, err = db.Open(Adapter, settings); err != nil {
+		t.Fatal(err)
+	}
+
+	defer sess.Close()
+
+	b := sess.Builder()
+
+	// SELECT * FROM artist
+	res = b.Select().From("artist")
+
+	if err = res.One(&artist); err != nil {
+		t.Fatal(err)
+	}
+
+	// SELECT * FROM artist WHERE name LIKE '%F%'
+	res = b.Select().From("artist").Where("name LIKE ?", "%F%")
+
+	if err = res.One(&artist); err != nil {
+		t.Fatal(err)
+	}
+
+	// SELECT "id" FROM artist WHERE name LIKE '%Miya%' OR name LIKE 'F%'
+	res = b.Select("id").From("artist").Where(`name LIKE ? OR name LIKE ?`, "%Miya%", "F%")
```

Comment:
Interesting, so the Where() condition takes an `...interface{}` and supports taking a string with arguments on the tail.. or a db.Cond{} map..? anything else?